managed/ui/src/components/tables/ListBackups/ListBackups.js

```diff
@@ -366,8 +366,8 @@ export default class ListBackups extends Component {
       .map((b) => {
         const backupInfo = b.backupInfo;
         if (
-          (backupInfo.actionType === 'CREATE' && backupInfo.status !== 'Deleted') ||
-          (showDeletedBackups && backupInfo.status === 'Deleted')
+          (backupInfo.actionType === 'CREATE' && b.state !== 'Deleted') ||
```

Comment:
backupInfo.status for Deleted is unknown. So required to change to b.state(line number 273, we are assigning status value
).